cognite/extractorutils/rest/extractor.py

```diff
@@ -459,3 +460,19 @@ def recursive_get_or_call(item: RequestBodyTemplate) -> RequestBody:
             return res
 
     return json.dumps(recursive_get_or_call(body))
+
+
+def _incompatible_lists_to_dict(data: Any) -> Dict:
+    return _incompatible_lists_to_dict_recursive(data)
+
+
+def _incompatible_lists_to_dict_recursive(data: Any, parent: Any = None) -> Any:
```

Comment:
Does dacite not handle `List[List[...]]`? I see how it is hard to represent a list directly, but is the same true for nested lists?

Reply:
Hmm, good point! I'll run some experiments and check

Reply:
It does inedeed handle nested lists, I'll change it to only convert a list at the root of the JSON structure.